.github/workflows/bindings_nodejs.yml

```diff
@@ -68,8 +66,8 @@ jobs:
       - name: Check diff
         run: git diff --exit-code
 
-      - name: Unit test
-        run: cargo nextest run --no-fail-fast
```

Comment:
https://github.com/apache/incubator-opendal/actions/runs/6753573761/job/18360177973?pr=3484
Can you please help me to find out what configuration is missing from the compilation? 
I don't have an environment to be able to test it.
It's wired that running a command test will result in missing libs that fail to compile, but building it directly won't.

cc @messense

Reply:
Seems napi-rs error, FYI https://github.com/napi-rs/napi-rs/issues/1782

Reply:
> Seems napi-rs error, FYI https://github.com/napi-rs/napi-rs/issues/1782

Thanks for raising the issue. 😊